modules/installation-osp-dpdk-binding-vfio-pci.adoc

```diff
@@ -13,145 +13,178 @@ Compute machines that connect to a virtual function I/O (VFIO) network require t
 $ openstack network show <VFIO_network_name> -f value -c id
 ----
 
-. Create a machine set on your cluster from the following template:
+. Create a Butane config file, `99-vhostuser-bind.bu`, binding the PCI device to the VFIO driver.
++
+[NOTE]
+====
+See "Creating machine configs with Butane" for information about Butane.
```

Comment:
Link for this?

Reply:
I looked for other references of Butane in this repo and none of them have a link for reference. Please suggest if you want me to change this content.

Reply:
It's nice when possible, but we cannot link from one module to another due to publishing toolchain limitations. :\ 

I couldn't tell from context alone if this was in the same docs set or not. Since it is, possible options are:

* Create an "Additional resources" section after this module in any assemblies that contain it. A link out can live there. 
* Add more context to the note to help with search. Ex: "For more information about Butane, see 'Creating machine configs with Butane' in the 'Customizing nodes' section of the documentation."  
* Leave it as is.

Reply:
I went with option 2, let me know if it's good.